tests/ignite/contrib/handlers/test_custom_events.py

```diff
@@ -8,55 +8,58 @@
 
 def test_bad_input():
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations="a")
+    with pytest.warns(DeprecationWarning):
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations="a")
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=0)
+        with pytest.raises(ValueError):
```

Comment:
Does it work with a single `pytest.warns` to catch multiple warnings ?

Reply:
It seems it does, but that test looked like this before this PR